cordova-lib/spec-cordova/platform.spec.js

```diff
@@ -356,3 +356,123 @@ describe('plugin add and rm end-to-end --fetch', function () {
         .fin(done);
     }, 60000);
 });
+
+describe('cocoapod plugin add and rm end-to-end', function () {
+
+    var tmpDir = helpers.tmpDir('cocoapod_plugin_test');
+    var project = path.join(tmpDir, 'hello4');
+
+    var samplePlugin = path.resolve('./spec-cordova/fixtures/plugins/sample-cordova-plugin-cocoapod-dependent');
+    var overlappingDependencyPlugin = path.resolve('./spec-cordova/fixtures/plugins/sample-cocoapod-plugin-overlapping-dependency');
+    var AFNetworking = 'AFNetworking',
+        CWStatusBarNotification = 'CWStatusBarNotification';
+    var podfile, podsJSON, workspace;
+
+    beforeEach(function() {
+        shell.exec('pwd');
+        process.chdir(tmpDir);
+    });
+
+    afterEach(function() {
+        process.chdir(path.join(__dirname, '..'));  // Needed to rm the dir on Windows.
+        shell.rm('-rf', tmpDir);
+    });
+
+    it('installs and uninstalls plugin depending on new pod and existing pod', function(done) {
+
+        cordova.raw.create('hello4')
+        .then(function() {
+            process.chdir(project);
+            //TODO: change this to cordova-ios on npm 
+            return cordova.raw.platform('add', 'https://github.com/juliascript/cordova-ios.git#CB-9825');
```

Comment:
Todo: change this to cordova-ios on npm once new cordova-ios is published with this pr. 

Thought I would add your todo to the pr comments